### PR TITLE
add ios uiautomation driver doc

### PR DESCRIPTION
**Pull request by jlipps**
and delete/consolidate some older docs

### CODE REVIEW

File: docs/en/drivers/ios-uiautomation.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+The way to start a session using the UIAutomation driver is to set the
+`platformName` [capability](#TODO) in your [new session request](#TODO) to the
+value of `iOS`. Of course, you must also include appropriate `platformVersion`,
+`deviceName`, and `app` capabilities, at a minimum.
+
+### Capabilities
+
+The UIAutomation driver supports a number of standard [Appium
+capabilities](/docs/en/writing-running-appium/caps.md), but has an additional
+set of capabilities that work for this driver only (see the "iOS" section of
```
Comment (imurchie) [inline docs/en/drivers/ios-uiautomation.md]: I presume the capabilities doc will still have sections that are linkable, so why not link directly there?

File: docs/en/drivers/ios-uiautomation.md [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+> **Note**: This driver is _DEPRECATED_ and should not be used unless
+> absolutely necessary. The information in this doc may not keep up to date
+> with reality, and the driver will be removed in a future version of Appium.
+> To begin iOS automation with Appium today, please use the [XCUITest
+> Driver](/docs/en/drivers/ios-xcuitest.md) instead.
+
+Appium's former method for iOS app automation was based on `UIAutomation`, an
+Apple-provided framework that shipped with the iOS SDK until iOS 10, when it
+was removed. `UIAutomation` was one of the tools included in Apple's
+`instruments` profiling system, and provided a JavaScript API that ran
```
Comment (imurchie) [inline docs/en/drivers/ios-uiautomation.md]: Elsewhere "Instruments" is capitalized.

File: docs/en/drivers/ios-uiautomation.md [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+1. By default, Instruments-based automation is limited by the inclusion of
+   a 1-second hard-coded delay between commands, implemented for obscure
+   reasons by Apple's engineers. There is a way around this limitation called
+   [instruments-without-delay](https://github.com/facebookarchive/instruments-without-delay)
+   (IWD). IWD ships with Appium for Xcode versions < 7. For 7.x and up, IWD
+   must be installed manually by the user in advance of using Appium. The way
+   to do this is as follows:
+
+    * Clone the [appium-instruments](https://github.com/appium/appium-instruments) repository
```
Comment (imurchie) [inline docs/en/drivers/ios-uiautomation.md]: This doesn't exist anymore. The script is now in https://github.com/appium/appium-ios-driver/blob/master/bin/xcode-iwd.sh

File: docs/en/drivers/ios-uiautomation.md [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+empty and instead set the `browserName` capability to `Safari`.
+
+### Commands
+
+To see the various commands Appium supports, and specifically for information
+on how the commands map to behaviors for the UIAutomation driver, see the [API
+Reference](#TODO).
+
+### Simulator Setup
+
```
Comment (imurchie) [inline docs/en/drivers/ios-uiautomation.md]: Perhaps a short bit about the fact that only a single Simulator can be running, and automated, at a time?

File: docs/en/drivers/ios-uiautomation.md [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+Running tests on real devices is considerably more complicated due to code
+signing and additional workarounds to Apple limitations. The basic process for
+a successful automation strategy using this driver are as follows:
+
+1. Build your app with a Debug configuration, for the specific type of real
+   device you will run the test on, ensuring that the app is also signed for
+   running on your specific device. For example:
+
+    ```
+    xcodebuild -sdk <iphoneos> -target <target_name> -configuration <Debug> \
```
Comment (imurchie) [inline docs/en/drivers/ios-uiautomation.md]: `-configuration Debug` does not need the angle brackets.

**Review by imurchie — CHANGES_REQUESTED** [review]
A few little changes. Otherwise awesome!

**Comment by jlipps** [comment]
Thanks for the feedback, I've pushed some changes

**Pull request merged**